src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -64,6 +73,110 @@ def create_resource_group(rg_name, location, tags=None):
     )
     return rcf.resource_groups.create_or_update(rg_name, parameters)
 
+def create_appliance(resource_group_name, appliance_name, managed_rg_id, location, kind, appliance_definition_id=None, plan_name=None, plan_publisher=None, plan_product=None, plan_version=None, tags=None, parameters=None):
```

Comment:
appliance - we made a decision call this "managed application" - can we change the cmdlet to create_managed_app